demo-app/src/app/ot/playground/page.tsx

```diff
@@ -1,12 +1,50 @@
-import { parseYaml, applyConfiguration } from '@open-truss/open-truss'
```

Comment:
After my [playground updates PR](https://github.com/open-truss/open-truss/pull/57) merges, what do you think about making this into a demo-app OT component and have it rendered with a new config? 

e.g. you can make an `demo-app/src/open-truss/components/trino-client-demo.tsx` component and config called `demo-app/src/open-truss/configs/2-trino-client-demo.yaml` to the effect of.

```yaml
workflow:
  version: 1
  frames:
    - frame:
      view:
        component: TrinoClientDemo
```